### PR TITLE
New packages: kdb, kproperty, kreport

### PR DESCRIPTION
**Pull request by a17r**
In kdb, so far only sqlite is actually built, but mysql and postgres should be enabled soon.

kde-apps/kdb[sqlite] looks like a dependency of kexi
kde-apps/kreport is going to be a dependency of plan

### CODE REVIEW

File: kde-apps/kdb/kdb-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit kde5 python-single-r1
+
+DESCRIPTION="A database connectivity and creation framework for various database vendors"
```
Comment (johu) [inline kde-apps/kdb/kdb-9999.ebuild]: Description looks a bit long to me.

File: kde-apps/kdb/kdb-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
```
Comment (johu) [inline kde-apps/kdb/kdb-9999.ebuild]: Does it only work with py2.7 or is it maybe possible to build ith wit py3.x?

File: kde-apps/kproperty/kproperty-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit kde5
+
+DESCRIPTION="A property editing framework with editor widget similar to what is known from Qt Designer"
```
Comment (johu) [inline kde-apps/kproperty/kproperty-9999.ebuild]: Description looks to long to me. "A " can be removed and maybe "to what is known from" replaced with "like".

File: kde-apps/kdb/kdb-9999.ebuild (hunk 2 above)
Comment (a17r) [inline kde-apps/kdb/kdb-9999.ebuild]: Unfortunately doesn't work with python 3, i.e. still using print statement

File: kde-apps/kproperty/kproperty-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+RDEPEND="
+	$(add_frameworks_dep kconfig)
+	$(add_frameworks_dep kcoreaddons)
+	$(add_frameworks_dep kguiaddons)
+	$(add_frameworks_dep kwidgetsaddons)
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+"
+
+DEPEND="${RDEPEND}"
```
Comment (johu) [inline kde-apps/kproperty/kproperty-9999.ebuild]: DEPEND == RDEPEND then DEPEND should contain the deps.

File: kde-apps/kreport/kreport-9999.ebuild [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+KDE_TEST="true"
+inherit kde5
+
+DESCRIPTION="A framework for the creation and generation of reports in multiple formats"
```
Comment (johu) [inline kde-apps/kreport/kreport-9999.ebuild]: Same as the other descriptions.

File: kde-apps/kreport/kreport-9999.ebuild [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+	$(add_frameworks_dep kwidgetsaddons)
+	$(add_frameworks_dep kross)
+	$(add_kdeapps_dep kproperty)
+	dev-qt/qtgui:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+"
+
+DEPEND="${RDEPEND}"
```
Comment (johu) [inline kde-apps/kreport/kreport-9999.ebuild]: Same as the other comment with DEPEND==RDEPEND.

**Comment by johu** [comment]
Thanks. Rebased onto kde overlay master.

**Pull request closed without merging**